package.json

```
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/tooolbox/node-potrace#readme",
   "dependencies": {
-    "jimp": "^0.6.4"
+    "jimp": ">=0.9.6 <1"
```

I'd argue that this is a dangerous assumption that nothing breaking will hit between these versions. There could be a `0.10.0` or something. I'd suggest keeping it at:
```suggestion
    "jimp": "^0.9.6"
```

---

My thinking was that this would be similar to allowing minors using the caret with non-zero majors.

But yeah, i read [the semver spec](https://semver.org/#spec-item-4) and I guess these "Major version 0" packages are considered unstable ☹️ 

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

Accepting your suggestion.